sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -218,6 +228,15 @@ public ByteBuffer body() {
         return body.duplicate();
     }
 
+    /**
```

Comment:
Since we're updating this API surface area. Would you be opposed to adding this surface area to the EventData of yours, too? @jsquire @ramya-rao-a @YijunXieMS

Reply:
In Javascript, we allow users to pass in any type of data for the `body`.
We have a default encode/decode behavior which the user can override as well.

Reply:
I've no objection, and think there's value in this.   That said, we need to have a consensus around the naming for it.     ` BodyAsString` feels on the awkward side to me, though I can't think of anything that I like better.

@ramya-rao-a:  As the arbiter of all things Event Hubs, is this the name that we should go with, or do you want to raise it up before we officially take the change?

Reply:
@ramya-rao-a :  If we're taking this across the board, could we either have confirmation in a Teams thread with the requested API shape or (preferably) in an issue for each language so that we can plan for it?